helm-chart/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.2
+version: 0.5.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v0.4.2"
+appVersion: "v0.5.0"
```

Comment:
@NimJay, I see that `onlineboutique-0.5.0.tgz` is commited, we should revert that file (and add it in the `.gitignore` file somewhere).

Reply:
Good eye! And thank you for the thorough review (i.e., for double checking the file changes)!